git-hg/updatetags8.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+#
+# ===========================================================================
+# (c) Copyright IBM Corp. 2018 All Rights Reserved
+# ===========================================================================
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
+#
+# ===========================================================================
+
+if [ $# -lt 2 ]; then
+   echo Usage: $0 '[jdk8u|jdk9] TAGS'
+   echo Version supplied should match a reposotiry in adoptopenjdk/openjdk-VERSION
+   echo And also http://hg.openjdk.java.net/VERSION/VERSION
+   exit 1
+fi
+[ -z "$WORKSPACE" -o ! -d "$WORKSPACE" ] && echo Cannot access \$WORKSPACE: $WORKSPACE && exit 2
+
+set -euo pipefail
+
+MODULES=(corba langtools jaxp jaxws nashorn jdk hotspot)
+
+# These variables needs to define in Jenkin's job pipeline
+OPENJDK_VERSION="$1"
+TAGS="$2"
+GITHUB_PROJECT=git@github.com:AdoptOpenJDK
+GITHUB_REPO="openjdk-$OPENJDK_VERSION"
+
+# Clean up
+rm    -rf $WORKSPACE/$GITHUB_REPO $WORKSPACE/openjdk
+mkdir -p  $WORKSPACE/$GITHUB_REPO $WORKSPACE/openjdk/mirror 
+
+git --version || exit 1
+GIT_VERSION=$(git --version | awk '{print$NF}')
+GIT_MAJOR_VERSION=$(echo $GIT_VERSION | cut -d. -f1)
+GIT_MINOR_VERSION=$(echo $GIT_VERSION | cut -d. -f2)
+[ $GIT_MAJOR_VERSION -eq 1 ] && echo I need git version 2.16 or later and you have $GIT_VERSION && exit 1
+[ $GIT_MAJOR_VERSION -eq 2 -a $GIT_MINOR_VERSION -lt 16 ] && echo I need git version 2.16 or later and you have $GIT_VERSION && exit 1
+# https://raw.githubusercontent.com/felipec/git-remote-hg/master/git-remote-hg
+if ! which git-remote-hg 2>/dev/null; then
+  echo I need git-remote-hg and could not find it
+  echo Get it from http://raw.githubusercontent.com/felipec/git-remote-hg/master/git-remote-hg
+  exit 1
+fi
+
+# Clone current AdoptOpenJDK repo
+cd $WORKSPACE/$GITHUB_REPO
+echo "Clone current $GITHUB_REPO"
+git clone $GITHUB_PROJECT/$GITHUB_REPO.git
+cd $GITHUB_REPO
+git fetch --tags
+OLDTAG=$(git describe --abbrev=0 --tags) 
+echo "Current openjdk level is $OLDTAG"
+
+# Clone current openjdk from Mercurial
+echo "Get base openjdk repository" 
+cd $WORKSPACE/openjdk/mirror
+git init
+git clone --bare "hg::http://hg.openjdk.java.net/$OPENJDK_VERSION/$OPENJDK_VERSION"
```

Comment:
Not convinced that's the right URL for 9, see previous comment

Reply:
Fixed